TunnelVisionLabs.ReferenceAssemblyAnnotator/Program.cs

```diff
@@ -93,6 +106,16 @@ private static void AnnotateType(TaskLoggingHelper? log, TypeDefinition typeDefi
             }
 
             Annotate(typeDefinition, annotatedTypeDefinition, attributesOfInterest);
+            for (var i = 0; i < typeDefinition.Interfaces.Count; i++)
+            {
+                for (int j = 0; j < annotatedTypeDefinition.Interfaces.Count; j++)
```

Comment:
One is `var` and one is `int`.

Reply:
Fixed and also filed https://github.com/dotnet/roslyn/issues/38953